model/src/pyrenew/model/hospitalizations.py

```diff
@@ -87,7 +88,7 @@ def __init__(
             gen_int=gen_int,
             I0=I0,
             latent_infections=latent_infections,
-            observed_infections=DeterministicVariable((0,)),
+            observed_infections=DeterministicVariable(0),
```

Comment:
Yes, I got confused by the name myself! I agree that none should be a better option. I'm pushing an update to reflect that in a few minutes. The thing about the NullObservationProcess is that it would return None from the sample. I think that's the cleanest.

Reply:
Partly addressed now, opened an issue to deal with more general cases across the package #85.

Reply:
PR #93 covers this now.